app/src/components/features/rules/RulePairs/Pairs/Rows/RowsMarkup/DestinationURLRow/index.js

```diff
@@ -163,7 +202,25 @@ const DestinationURLRow = ({
         }}
       >
         <Col span={3}>
-          <span>Redirect to</span>
+          <span className="redirect-rule-destination-label">
+            Redirect to{" "}
+            {showInfoToolTipWithABTest && (
+              <Tooltip
+                title={
+                  "Define the destination URL where you want to redirect the original request."
+                }
+                onOpenChange={(open) => {
+                  if (open)
+                    trackMoreInfoViewed(
+                      currentlySelectedRuleConfig.TYPE,
+                      "destination_url"
+                    );
+                }}
+              >
+                <img src={InfoOutlinedIcon} alt="info" />
+              </Tooltip>
```

Comment:
AFAIU, there are 2 cases to get more info on an element:
1. Show an info icon next to the element. Hovering/clicking on the info icon should open the tooltip.
2. Hovering on the element itself should open the tooltip.

Can we create a reusable component and use like this:
```js
<MoreInfo
  text="Define the destination URL..."
  showIcon
  analyticsContext="Redirect_destination_url"
>
  Redirect to
</MoreInfo>
```

`MoreInfo` element can get the element using `children` prop and add tooltip on the element or the info icon.